/detect_aruco_marker_action_server:
  ros__parameters:
    sub_marker_topic: "/marker"
    required_pose_num: 2
    sub_rate: 10
    timeout: 5000
